code/modules/clothing/shoes/heels.dm

```diff
@@ -32,6 +32,10 @@
 #define TRIP_CHANCE_INCREASE 0.5
 
 /obj/item/clothing/shoes/heels/handle_movement(turf/walking, running)
+	if(running)
+		playsound(src, SFX_HEELS, 40, 1) // this will get annoying very fast.
+	else
+		playsound(src, SFX_HEELS, 20, 1)
```

Comment:
```suggestion
	THROTTLE(cooldown, 1 SECOND)
	if(!cooldown)
		return

	if(running)
		playsound(src, SFX_HEELS, 40, 1) // this will get annoying very fast.
	else
		playsound(src, SFX_HEELS, 20, 1)
```